nss-day-cohort-46/rare-rest-rare-api-cool-cake
# Pp create comment

Description:
Backend code for creating comments is complete.

Testing Instructions:

Postman URL:
```
http://localhost:8000/comments
```

Postman Body:
```
{
    "postId": 1,
    "content": "new comment",
    "createdOn": "2016-05-18T15:37:36.993048Z"
}
```

Make a POST request and check database to see if a new comment was added.
